input/pagecontent/authentication.md

```diff
@@ -0,0 +1,279 @@
+### Introduction
+
+Authentication is the process of verifying the identity of an entity. In the context of healthcare, authentication is used to verify the identity of healthcare professionals and care organizations. This is important to ensure that only authorized entities can access sensitive information and resources.
+
+The verified identity is often used downstream for authorization and accounting. Authorization is the process of granting access to resources based on the identity of an entity. Accounting is the process of tracking the actions of an entity.
+
+#### Requirements
+
+There exists a multitude of authentication solutions, therefore it is important to define the requirements for authentication in the context of healthcare. The requirements are:
+
+- Work with identity claims from the authoritative sources
```

Comment:
using/compliance to established standards (e.g. OAuth2 or whatever) isn't a requirement?